### DISCUSSION TITLE
NZ:P Community Discord Server Link

### DISCUSSION THREAD

**Discussion by crypticalwastaken** [nzp-team/nzportable#516] [General]
For people having a hard time finding this and don't feel like searching the entire internet just to find it: https://discord.gg/6GBGf4N

**Comment by crypticalwastaken** [comment]
I strongly enjoy this project and would hate to see it die because of lack of interest.

**Comment by MotoLegacy** [comment]
Hi, just letting you know the project does not associate with this server anymore. I don't condone or endorse any of the discussions that occur there. Hence why the links have been removed from official channels. Thanks.

**Reply by crypticalwastaken** [reply]
Where's the official server then?

**Reply by crypticalwastaken** [reply]
You expect people to play this game but don't want a place where they can converse about map making, high scores, as well as setup game nights. This is stuff that has been going on since the days of quake.

**Reply by MotoLegacy** [reply]
Why are you making nasty assumptions? That's just plain rude. It's recommended to have discussions about that stuff here so it can be properly and categorically logged. The GitHub Discussions page you're on now has categories dedicated to general banter, map records and speedruns, map maker/mod releases, etc. All sorted in such a way to where someone can easily find something specific. You don't get that with Discord. I would appreciate you not assuming I'm not giving people recommended spaces to communicate about the game, in the exact space created for people to communicate about the game.

**Reply by crypticalwastaken** [reply]
Most people don't even know what github is. Also you could literally do the same thing you're doing here with threads on discord. as for my tone i apologize but this is truly how you kill a community.

**Reply by crypticalwastaken** [reply]
Audit logs also exist on discord as well.

**Reply by MotoLegacy** [reply]
No, doing that requires people to register an account on Discord just to view that content. Discord "forums" also alienate people who don't use the vanilla clients — whatever reason that may be (NZ:P targets low spec hardware, it's not unreasonable to suggest the userbase who also use Discord may also not be capable of running the web client). GitHub is the place to download the game, it's linked absolutely everywhere officially. How is that alienation? Not being the platform you want doesn't automatically make it a problem, especially when it's advertised everywhere, and is way more accessible. You're really being disrespectful.

**Comment by Naievil** [comment]
I really appreciate the discussion, but I just feel it is best to lock the thread. Sorry all.
